app/models/eto_api/base.rb

```diff
@@ -275,9 +281,9 @@ def entity_sub_types(entity_type_id:, site_id:)
     end
     memoize :entity_sub_types
 
-    def entities_by_entity_type_id(entity_type_id:, entity_subtype_id:, programId:, site_id:)
+    def entities_by_entity_type_id(entity_type_id:, entity_subtype_id:, programId:, site_id:) # rubocop:disable Naming/MethodParameterName, Naming/VariableName
```

Comment:
Why not rename `programId`?

Reply:
Only because we'd have to track down all the calls to `entities_by_entity_type_id`.  There may not be many.  We can come back and cleanup `rubocop:disable` lines after we have coverage across all models.